user_tools/docs/user-tools-databricks-aws.md

```diff
@@ -261,26 +260,26 @@ argument `--remote_folder` was a valid S3 path.
 
 A cluster property is still accessible if one of the following conditions applies:
 
-1. The cluster is listed by the `databricks clusters get --cluster-name $CLUSTER_NAME` cmd. In this case, the CLI will be triggered by providing
-   `--gpu_cluster $CLUSTER_NAME`
+1. The cluster is listed by the `databricks clusters get CLUSTER_ID [flags]` cmd. In this case, the CLI will be triggered by providing
```

Comment:
This comes from `databricks clusters get -h`, which prints:
```
...
Usage:
  databricks clusters get CLUSTER_ID [flags]

Flags:
...
```
I am ok to remove them. I kept them here because sometime users may have different profiles for databricks-aws and databricks-azure and in that case we need to add `--profile` to get the correct cluster info (and keeping the `[flags]` here could be a reminder). But I am not sure if this is common doc style so I am open to discuss this and just keep `databricks clusters get CLUSTER_ID` everywhere. Thanks! @parthosa

Reply:
That makes sense. Thanks @cindyyuanjiang